onnxruntime/core/optimizer/qdq_transformer/qdq_s8_to_u8.h

```diff
@@ -16,11 +16,15 @@ namespace onnxruntime {
 class QDQS8ToU8Transformer : public GraphTransformer {
  public:
   QDQS8ToU8Transformer(bool weights_to_u8, const InlinedHashSet<std::string_view>& compatible_execution_providers = {}) noexcept
-      : GraphTransformer("QDQS8ToU8Transformer", compatible_execution_providers), weights_to_u8_(weights_to_u8) {}
+      : GraphTransformer("QDQS8ToU8Transformer", compatible_execution_providers), weights_to_u8_(weights_to_u8) {
+          exclude_next = {"MatMul", "Conv", "Gemm", "Gather"};
+      }
+  bool ShouldConvertWeightFromS8ToU8(Graph& graph, Node& node) const;
 
  private:
   Status ApplyImpl(Graph& graph, bool& modified, int graph_level, const logging::Logger& logger) const override;
   bool weights_to_u8_;
+  std::set<std::string> exclude_next;
```

Comment:
Please replace std::set with InlinedHashSet, which allocates a single buffer for all of its keys and values that avoids frequent per-node allocations: https://github.com/microsoft/onnxruntime/blob/0c6037b5abe571fc43a55ef7a9d2f846820fbe5d/include/onnxruntime/core/common/inlined_containers.h#L50

Reply:
ORT adds a "_" for member variable. for the naming itself, how about operators_prefer_S8_?